interfaces/PostGoal.ts

```diff
@@ -0,0 +1,5 @@
+export interface PostGoal {
```

Comment:
didn't we require time duration(in which they complete the task)  field ?

Reply:
you have also integrated the API as well, will you please add a description as your ticket doesn't have anything regarding the api integration?

Reply:
There is `Goal` & `UserGoal`.

The UserGoal have things like, completion duration, status, derferUntil etc